service/src/test/resources/application-test.yml

```diff
@@ -39,3 +40,9 @@ terra.common:
     in-kubernetes: false
   tracing:
     stackdriverExportEnabled: false
+
+workspace:
```

Comment:
Needed for spring to be able to instantiate the config classes.
Existing tests are using the same config, they just aren't relying on spring to instantiate it.

Reply:
These credentials should not be present in git, please remove.